buildbot_effects/__init__.py

```diff
@@ -222,13 +222,20 @@ def run_effects(
         "/proc",
         "--dev",
         "/dev",
+        "--uid",
+        "0",
+        "--gid",
+        "0",
```

Comment:
What is the default? Should we not just keep the uid rather than pretending to be root? Some applications might than behave funny.

Reply:
Default seems to be root:

https://github.com/hercules-ci/hercules-ci-effects/blob/b89ac4d66d618b915b1f0a408e2775fe3821d141/effects/effect/effect.nix#L79

Reply:
Ok than.